src/app/app.component.html

```diff
@@ -1,9 +1,9 @@
-<app-header></app-header>
+<app-header (menuChangeEvt)="changeMunu($event)"></app-header>
 <div class="container">
   <div class="row">
     <div class="col-md-12">
-      <app-recipes></app-recipes>
-      <app-shopping-list></app-shopping-list>
+      <app-recipes *ngIf="menuSelector === 'recipes'"></app-recipes>
```

Comment:
I would prefer switch here since you need to choose between several cases
[ngSwitch]="activeTab"

"menuSelector" I suggest to rename. For example:  activeTab. menuSelector variable does not select anything. It's just storage. That's why name is quite confusing